examples/typescript-angular/js/Application.js

```diff
@@ -53,6 +53,28 @@ var todos;
 })(todos || (todos = {}));
 /// <reference path='../_all.ts' />
 var todos;
+(function (todos) {
+    'use strict';
+    var ESCAPE_KEY = 27;
```

Comment:
Although the change will be reflected in `TodoEscape.ts` down the diff, this is the compiled file.